lib/webauthn/configuration.rb

```diff
@@ -27,19 +29,28 @@ def self.if_pss_supported(algorithm)
     attr_accessor :verify_attestation_statement
     attr_accessor :credential_options_timeout
     attr_accessor :silent_authentication
+    attr_accessor :acceptable_attestation_types
+    attr_reader :attestation_root_certificates_store
 
     def initialize
       @algorithms = DEFAULT_ALGORITHMS.dup
       @encoding = WebAuthn::Encoder::STANDARD_ENCODING
       @verify_attestation_statement = true
       @credential_options_timeout = 120000
       @silent_authentication = false
+      @acceptable_attestation_types = [:None, :Self, :Basic]
     end
 
     # This is the user-data encoder.
     # Used to decode user input and to encode data provided to the user.
     def encoder
       @encoder ||= WebAuthn::Encoder.new(encoding)
     end
+
+    def attestation_root_certificates_store=(store)
```

Comment:
I found this name to be quite confusing since it's named the same as the method in `AuthenticatorAttestationResponse` - a name along the lines of `attestation_root_certificates_store_{finder,resolver}` would help distinguish the two.

Reply:
I see the confusion. Considering this and your other comment above regarding the trust store creation on `AuthenticatorAttestationResponse`, I think I'll go with the following:
- Drop the word 'store' on the configuration option to avoid confusion between a store in the sense of a structure capable of storing certificates and an `OpenSSL::X509::Store`. I'll take part of your suggestion and probably rename it to something like `attestation_root_certificates_finder`. The object used here has to implement a `find` method that returns an array containing the certificates for a certain format/authenticator, same as now.
- `AuthenticatorAttestationResponse#attestation_root_certificates_store` will remain with that name but will return an `OpenSSL::X509::Store` created with the certificates obtained using the configuration option above.

Do you think that'll be clearer?